### ISSUE TITLE
circleci: failed tests: TestDockerFinagle 

### ISSUE THREAD

**Issue by tschottdorf** [cockroachdb/cockroach#6014]
The following test appears to have failed:

[#16204](https://circleci.com/gh/cockroachdb/cockroach/16204):

```
	at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

FAILURES!!!
Tests run: 12,  Failures: 1

I160412 17:38:54.036480 acceptance/cluster/localcluster.go:649  stopping
I160412 17:38:54.036740 acceptance/cluster/localcluster.go:547  event stream done, resetting...: net/http: request canceled
I160412 17:38:54.036760 acceptance/cluster/localcluster.go:570  events monitor exits
I160412 17:38:54.037932 acceptance/cluster/docker.go:397  ContainerKill: *errors.errorString: Error response from daemon: Cannot kill container 5bb6a97dea31b83e9016c107ff6d6a521b9a71b09c4400300aa832e64c947e2c: Container 5bb6a97dea31b83e9016c107ff6d6a521b9a71b09c4400300aa832e64c947e2c is not running
--- FAIL: TestDockerFinagle (12.66s)
	util_test.go:281: expected success: non-zero exit code: 1
=== RUN   TestGossipPeerings
I160412 17:38:56.624147 acceptance/cluster/localcluster.go:286  Initializing Cluster AdHoc 3x1:
{"name":"AdHoc 3x1","nodes":[{"count":3,"stores":[{"count":1,"max_ranges":0}]}],"duration":5000000000}
I160412 17:38:56.625837 acceptance/cluster/docker.go:102  ImagePull cockroachdb/builder:20160305-182433 already exists
I160412 17:38:57.145484 acceptance/cluster/localcluster.go:581  creating certs (1024bit) in: /home/ubuntu/cockroach/.localcluster.certs.387676759
I160412 17:38:57.203709 acceptance/cluster/localcluster.go:532  events monitor starts
I160412 17:38:57.636123 acceptance/cluster/localcluster.go:480  *** started roach0 ***
  ui:        https://127.0.0.1:32826
  trace:     https://127.0.0.1:32826/debug/requests
--
--- SKIP: TestBuildBabyCluster (0.00s)
	terraform_test.go:30: only enabled during testing
=== RUN   TestFiveNodesAndWriters
--- SKIP: TestFiveNodesAndWriters (0.00s)
	util_test.go:65: running in docker mode
FAIL
ok github.com/cockroachdb/cockroach/acceptance 1337s

```
Please assign, take a look and update the issue accordingly.

**Comment by dt** [comment]
```
1) A postgres client should update a row(com.twitter.finagle.postgres.integration.IntegrationSpec)
com.twitter.finagle.postgres.codec.ServerError: PgRequest(Query(DROP TABLE IF EXISTS finagle_test),false)
SERRORCXX000Mstorage/store.go:1543: rejecting command with timestamp in the future: 1460482730813576847 (312.099364ms ahead)
```

**Comment by tschottdorf** [comment]
Hopefully fixed when #5845 lands.
